src/mixins/itext_click_behavior.mixin.ts

```diff
@@ -191,7 +192,10 @@ export abstract class ITextClickBehaviorMixin<
         return;
       }
     }
-
+    if (this.__isDragging && options.isClick) {
+      // false positive drag event, is actually a click
+      this.setCursorByClick(options.e);
+    }
```

Comment:
since `this.setCursorByClick` wasn't here before, who was taking care of it?

Reply:
I believe this is why you altered the logic in the original PR stating you changed something (forgot what)
Until now I missed what it did and why you (probably) did it.
Trying to click a selection does the following:
- mouse down before: set `__isDragging` to true
- mouse down: if `__isDragging === false` set cursor by click => **skipped**
- drag events fire or **not**, we are interested in the case they do not.
- mouse up: now we know if the interaction was a click or not so we can set cursor by click if `__isDragging` to compensate for what didn't happen in mouse down

Reply:
> I believe this is why you altered the logic in the original PR stating you changed something (forgot what)
Until now I missed what it did and why you (probably) did it.

Sorry I meant to link to it but now I think is simply unrelated

Reply:
I would add tests but it is quite a nightmare I believe since I need a sequence of drag events...
Am I being lazy? Might be.

Reply:
Maybe give me a few more hours and I try to write an extensive module of visual drag effects for the following cases:
- drag start
- drag over self
- drag over other target
- drag enter/leave canvas
- click

Doesn't sound awful actually